GoMeet/src/main/webapp/test/spec/spec-interactive-map.js

```diff
@@ -144,3 +144,67 @@ describe ('Build Info Window Vote', function() {
     expect(noteText).toBe(NOTE_A);
   });
 });
+
+/** Tests for displaying popular locations. */
+describe ('Display Popular Location', function() {
+  const LOCATIONS =
```

Comment:
You should add a location that isn't popular and doesn't have the same votecount as the others.

Reply:
Just to clarify, should I add a location with a lower vote count and expect it to be displayed as well given that fetching ```fetchPopularLocations()``` will return only the most popular locations?

Reply:
ahh whoops, my mistake. I misread these tests. What you have makes sense :) I forgot that we're essentially mocking out the servlet functionality and thought that we were testing it.

Reply:
Haha, np!